cfgov/v1/util/ref.py

```diff
@@ -334,3 +334,49 @@ def get_category_children(category_names):
             )
         )
     )
+
+
+def get_category_icon(category_name):
+    """Given a category name, return the name of its icon.
+
+    Return None if no matching icon exists.
+    """
+    return {
+        "at the cfpb": "bullhorn",
+        "auto loans": "car",
+        "bank accounts": "bank",
+        "bank accounts and services": "bank",
+        "blog": "speech-bubble",
+        "consumer advisories": "complaint",
+        "credit cards": "credit-card",
+        "credit reporting": "credit-report",
+        "credit reports and scores": "credit-report",
+        "data, research, and reports": "chart",
+        "debt collection": "debt-collection",
+        "director's notebook": "edit",
+        "director's statement": "open-quote",
+        "events": "date",
+        "families and money": "money",
+        "featured event": "date",
+        "featured blog": "speech-bubble",
+        "featured video": "play-round",
+        "featured tool": "settings",
+        "featured news": "newspaper",
+        "featured": "favorite",
+        "fraud and scams": "money",
+        "info for consumers": "information",
+        "mortgages": "mortgage",
+        "money basics": "money",
+        "money transfers": "money-transfer",
+        "newsroom": "newspaper",
+        "op-ed": "fountain-pen",
+        "payday loans": "payday-loan",
+        "policy and compliance": "bank",
+        "prepaid cards": "prepaid-cards",
+        "press release": "bullhorn",
+        "report": "chart",
+        "reverse mortgages": "loan",
+        "speech": "microphone",
+        "student loans": "college",
+        "testimony": "government",
```

Comment:
I think it's preferable if we only use the canonical icon name going forward and eventually phase out the aliases (per https://github.com/cfpb/design-system/issues/1252). In which case, this should be `taxes` instead of `government` according to https://cfpb.github.io/design-system/foundation/iconography

Reply:
@anselmbradford added your suggested change here.